backend/app/routers/auth.py

```
@@ -108,3 +108,32 @@
+            is_verified=user_info["email_verified"],
+            role="user",
+            password=None,
+        )
+
+        db.add(new_user)
+        db.commit()
+        db.refresh(new_user)
+        access_token = create_access_token({"sub": new_user.email})
+        return {"access_token": access_token, "token_type": "bearer"}
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)